notebooks/MNIST-torchvision.ipynb

```diff
@@ -0,0 +1,798 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  epoch    train_loss    valid_acc    valid_loss     dur\n",
+      "-------  ------------  -----------  ------------  ------\n",
+      "      1        \u001b[36m0.9299\u001b[0m       \u001b[32m0.9307\u001b[0m        \u001b[35m0.2432\u001b[0m  6.9704\n",
+      "      2        \u001b[36m0.3104\u001b[0m       \u001b[32m0.9539\u001b[0m        \u001b[35m0.1505\u001b[0m  7.0305\n",
+      "      3        \u001b[36m0.2220\u001b[0m       \u001b[32m0.9664\u001b[0m        \u001b[35m0.1168\u001b[0m  7.0733\n",
+      "      4        \u001b[36m0.1784\u001b[0m       \u001b[32m0.9713\u001b[0m        \u001b[35m0.0988\u001b[0m  7.0218\n",
+      "      5        \u001b[36m0.1550\u001b[0m       \u001b[32m0.9745\u001b[0m        \u001b[35m0.0882\u001b[0m  6.8893\n",
+      "      6        \u001b[36m0.1426\u001b[0m       \u001b[32m0.9753\u001b[0m        \u001b[35m0.0800\u001b[0m  6.8398\n",
+      "      7        \u001b[36m0.1304\u001b[0m       \u001b[32m0.9775\u001b[0m        \u001b[35m0.0731\u001b[0m  7.0749\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Exception ignored in: <bound method Image.__del__ of <PIL.Image.Image image mode= size=0x0 at 0x7F2D7F86AFD0>>\n",
+      "Traceback (most recent call last):\n",
+      "  File \"/home/marian/anaconda3/envs/skorch/lib/python3.6/site-packages/PIL/Image.py\", line 616, in __del__\n",
+      "    if (hasattr(self, 'fp') and hasattr(self, '_exclusive_fp')\n",
+      "KeyboardInterrupt\n"
+     ]
+    }
+   ],
+   "source": [
+    "cnn.fit(mnist_train, y=y_train);"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 117,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y_pred_cnn = cnn.predict(mnist_test)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 118,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.9815"
+      ]
+     },
+     "execution_count": 118,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "accuracy_score(y_test, y_pred_cnn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An accuracy of >98% should suffice for this example!\n",
+    "\n",
+    "Let's see how we fare on the examples that went wrong before:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 119,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.6682577565632458"
+      ]
+     },
+     "execution_count": 119,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "accuracy_score(y_test[error_mask], y_pred_cnn[error_mask])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Over 70% of the previously misclassified images are now correctly identified."
```

Comment:
This statement is incorrect for the current state of the notebook (possibly because training was interrupted). Perhaps change this to "The majority of ..."